frontend/packages/dev-console/src/components/helm/HelmReleasesDetailsPage.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { match as RMatch } from 'react-router';
+import * as _ from 'lodash';
+import { Firehose } from '@console/internal/components/utils';
+import { NamespaceBar } from '@console/internal/components/namespace';
+import HelmReleaseDetailsPage from './HelmReleaseDetailsPage';
+
+export interface HelmReleasesDetailsPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+}
+
+const HelmReleasesDetailsPage: React.FC<HelmReleasesDetailsPageProps> = ({ match }) => {
+  const namespace = match.params.ns;
+  const helmReleaseName = match.url.substring(_.lastIndexOf(match.url, '/') + 1);
```

Comment:
Why are you taking releaseName from substring? Why not make it a param and take it like namespace?